### PR TITLE
style: gofumpt and godot [skip changelog]

### PR DESCRIPTION
**Pull request by kehiy**
I added the `godot` and `gofumpt` linters to make a more readable code.

### CODE REVIEW

File: cmd/ipfs/init.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ const (
 	profileOptionName   = "profile"
 )
 
-// nolint
+// nolint.
```
Comment (hacdias) [inline cmd/ipfs/init.go:36]:
```suggestion
// nolint
```

**Review by hacdias — APPROVED** [review]
This makes me mostly happy. Thank you!

**Comment by Jorropo** [comment]
IMO it is worst to have inconsistent linters, it would be awesome if you would also submit it there https://github.com/ipfs/uci (this is a unified CI for our libraries across many repos). :slightly_smiling_face:

File: test/integration/wan_lan_dht_test.go [hunk 2]
```diff
@@ -51,8 +51,10 @@ func TestDHTConnectivitySlowRouting(t *testing.T) {
 }
 
 // wan prefix must have a real corresponding ASN for the peer diversity filter to work.
-var wanPrefix = net.ParseIP("2001:218:3004::")
-var lanPrefix = net.ParseIP("fe80::")
+var (
+	wanPrefix = net.ParseIP("2001:218:3004::")
+	lanPrefix = net.ParseIP("fe80::")
+)
```
Comment (Jorropo) [inline test/integration/wan_lan_dht_test.go:57]: Could we turn this one off ? This is not helping with readability and make copy-pastability worst and spread the code thin.

**Comment by hacdias** [comment]
@Jorropo yes, but they're not necessarily inconsistent. They're just stricter I would say. `godot` I like (didn't know, but I always hate seeing comments without dots in the end). `gofumpt` just seems stricter than `gofmt`.

File: repo/fsrepo/migrations/fetch.go [hunk 3]
```diff
@@ -184,7 +184,7 @@ func osWithVariant() (string, error) {
 //     "go-ipfs_v0.8.0-rc2_linux-amd64.tar.gz"
 //
 // This would form the path:
-// go-ipfs/v0.8.0/go-ipfs_v0.8.0_linux-amd64.tar.gz
+// go-ipfs/v0.8.0/go-ipfs_v0.8.0_linux-amd64.tar.gz.
```
Comment (Jorropo) [inline repo/fsrepo/migrations/fetch.go:187]: this is wrong

File: repo/fsrepo/fsrepo.go [hunk 4]
```diff
@@ -64,9 +64,11 @@ func (err NoRepoError) Error() string {
...
 }
 
-const apiFile = "api"
-const gatewayFile = "gateway"
-const swarmKeyFile = "swarm.key"
+const (
+	apiFile      = "api"
+	gatewayFile  = "gateway"
+	swarmKeyFile = "swarm.key"
+)
```
Comment (Jorropo) [inline repo/fsrepo/fsrepo.go:71]: similar for var, copy-pastability is worst when spread out.

File: p2p/p2p.go [hunk 5]
```diff
@@ -40,7 +40,7 @@ func New(identity peer.ID, peerHost p2phost.Host, peerstore pstore.Peerstore) *P
 }
 
 // CheckProtoExists checks whether a proto handler is registered to
-// mux handler
+// mux handler.
```
Comment (Jorropo) [inline p2p/p2p.go:43]: Looks wrong.

File: core/commands/routing.go [hunk 6]
```diff
@@ -495,8 +492,10 @@ identified by QmFoo.
 	Type: routing.QueryEvent{},
 }
 
-type printFunc func(obj *routing.QueryEvent, out io.Writer, verbose bool) error
-type pfuncMap map[routing.QueryEventType]printFunc
+type (
+	printFunc func(obj *routing.QueryEvent, out io.Writer, verbose bool) error
+	pfuncMap  map[routing.QueryEventType]printFunc
+)
```
Comment (Jorropo) [inline core/commands/routing.go:498]: similar to const and var

File: core/commands/routing.go [hunk 7]
```diff
@@ -21,9 +21,7 @@ import (
 	routing "github.com/libp2p/go-libp2p/core/routing"
 )
 
-var (
-	errAllowOffline = errors.New("can't put while offline: pass `--allow-offline` to override")
-)
+var errAllowOffline = errors.New("can't put while offline: pass `--allow-offline` to override")
```
Comment (Jorropo) [inline core/commands/routing.go:24]: Altho I think this is up to judgment to use braces blocks for declaration, I think this is good, if you have only one thing don't put it in a brace block.

File: core/commands/dag/dag.go [hunk 8]
```diff
@@ -13,9 +13,9 @@ import (
 	cid "github.com/ipfs/go-cid"
 	cidenc "github.com/ipfs/go-cidutil/cidenc"
 	cmds "github.com/ipfs/go-ipfs-cmds"
-	//gipfree "github.com/ipld/go-ipld-prime/impl/free"
-	//gipselector "github.com/ipld/go-ipld-prime/traversal/selector"
-	//gipselectorbuilder "github.com/ipld/go-ipld-prime/traversal/selector/builder"
+	// gipfree "github.com/ipld/go-ipld-prime/impl/free"
+	// gipselector "github.com/ipld/go-ipld-prime/traversal/selector"
+	// gipselectorbuilder "github.com/ipld/go-ipld-prime/traversal/selector/builder"
```
Comment (Jorropo) [inline core/commands/dag/dag.go:18]: This is on us, but WTF does commented out import in our code ?
Should be removed (in an other PR).

**Review by Jorropo — COMMENTED** [review]
I don't think this was in an acceptable state.

File: repo/fsrepo/migrations/fetch.go (hunk 3 above)
Comment (hacdias) [reply repo/fsrepo/migrations/fetch.go:187]: https://github.com/ipfs/kubo/pull/10082

File: core/commands/dag/dag.go (hunk 8 above)
Comment (hacdias) [reply core/commands/dag/dag.go:18]: https://github.com/ipfs/kubo/pull/10082

**Comment by Jorropo** [comment]
Actually I would like a linter that always don't use the braced declarations.

File: p2p/p2p.go (hunk 5 above)
Comment (hacdias) [reply p2p/p2p.go:43]: It's not? It's the end of a sentence.

**Comment by Jorropo** [comment]
Well I have a fork of gofumpt now https://github.com/ipfs/kubo/pull/10083

**Comment by Jorropo** [comment]
PS: I thought this PR added gofumpt to the CI, this was just a one shot. Things will degrade overtime if it's not enforced in CI FYI.

**Pull request merged**